TodoList.Web/Views/Home/Contact.cshtml

```diff
@@ -1,8 +1,4 @@
-﻿@{
-    ViewData["Title"] = "Contact";
```

Comment:
`ViewData["Title"]` will remain. because It is used as the tab's title.
![image](https://user-images.githubusercontent.com/17813786/66857351-c5d5dd80-ef8f-11e9-95d4-936707545af5.png)

Reply:
Doh! Fixed.